### PR TITLE
fix: add ignore pattern to import class

### PR DESCRIPTION
**Pull request by hanxiao**

### CODE REVIEW

File: tests/integration/test_helloworld.py [hunk 1]
```diff
@@ -105,7 +105,7 @@ def test_download_proxy(self):
         # head check
         req = urllib.request.Request(args.index_data_url, method="HEAD")
         response = urllib.request.urlopen(req, timeout=5)
-        self.assertEqual(response.status, 200)
+        assert response.status == 200
 
 
 if __name__ == '__main__':
```
Comment (JoanFM) [inline tests/integration/test_helloworld.py:111]: we can remove this here

File: tests/unit/drivers/querylang/test_querylang_reader.py [hunk 2]
```diff
@@ -65,7 +65,7 @@ def validate2(req):
 
     def test_querlang_driver(self):
         qld2 = dummyDriver(arg1='world')
-        self.assertEqual(qld2.arg1, 'world')
+        assert qld2.arg1 == 'world'
 
 
 if __name__ == '__main__':
```
Comment (JoanFM) [inline tests/unit/drivers/querylang/test_querylang_reader.py:71]: and here

Comment (hanxiao) [reply tests/unit/drivers/querylang/test_querylang_reader.py:71]: okay seems that my regex missed that

**Comment by hanxiao** [comment]
horray!

**Pull request merged**